pages/[gameId]/index.jsx

```diff
@@ -4,7 +4,10 @@ import GameDetailsView from "@/components/GameDetailsView";
 export default function GameDetailsPage({ games, onDeleteGame, onEditGame }) {
   const router = useRouter();
   const { gameId } = router.query;
-  let selectedGame = games.find((game) => game.id === gameId);
+  let selectedGame = games?.find(
+    (game) => game.id === gameId || game.id === Number(gameId)
+  );
```

Comment:
Would this not work just the same?

```suggestion
  let selectedGame = games?.find(
    (game) => game.id === Number(gameId)
  );
```

Reply:
The problem is that the id we get from the api is a number and the id we get from uid is a string. the uid is only used when we create a custom game. So I tried to cover both cases here.

Reply:
I see!

Then leave it the way it is 👌